framework/Samplers/Sampler.py

```diff
@@ -55,6 +55,7 @@ class cls.
         specifying input of cls.
     """
     inputSpecification = super(Sampler, cls).getInputSpecification()
+    # FIXME the DET HybridSampler doesn't use the "name" param for the samples it creates, so we can't require the name yet
```

Comment:
line is 124 columns... we should split this.

PS. Can you remember to give me a bit more details  in 30 mins about this thing?